bids/grabbids/bids_validator.py

```diff
@@ -252,3 +256,129 @@ def conditional_match(self, expression, path):
                 return False
         else:
             return False
+
+
+def validate_sequences(layout, config):
+    """Checks files in BIDS project match user defined expectations.
+
+    This method is a wrapper for the duplicate_check() and 
+    expected_file_check() methods. Use it to check whether there are
+    files with duplicate content within the BIDS data set and to check
+    the number of data set files against a user customized configuration
+    file. Returns three data frames: duplicate_file_df, summary_df, problem_df.
+
+
+    Parameters
+    ----------
+        layout: BIDSLayout class
+            A BIDSLayout path of a data set.
+
+        config: string
+            Path to customized configuration file.
+
+
+    Examples
+    --------
+    >>> layout = bids.grabbids.BIDSLayout('/path/to/sample/project_root')
+    >>> validate_sequences(layout, '/path/to/sample/config.json')
+    >>> duplicate_file_df
+    # Put example output here
+    >>> summary_df
+    # Put example output here
+    >>> problem_df
+    # Put example output here
+    """
+    
+    duplicate_file_df = duplicate_check(layout)
+    summary_df, problem_df = expected_file_check(layout, config)
+    return duplicate_file_df, summary_df, problem_df
```

Comment:
Instead of returning an ordinary tuple, you might want to return a `namedtuple` (import from `collections`), which allows you to add field names to tuples for more intuitive access (maybe name the fields `duplicates`, `summary`, and `problems`?).